doctor/ethtool/ethtool_linux.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package ethtool
+
+import (
+	"net/netip"
+
+	"github.com/safchain/ethtool"
+	"tailscale.com/net/interfaces"
+	"tailscale.com/types/logger"
+	"tailscale.com/util/set"
+)
+
+func ethtoolImpl(logf logger.Logf) error {
+	et, err := ethtool.NewEthtool()
+	if err != nil {
+		logf("could not create ethtool: %v", err)
+		return nil
+	}
+	defer et.Close()
+
+	interfaces.ForeachInterface(func(iface interfaces.Interface, _ []netip.Prefix) {
+		ilogf := logger.WithPrefix(logf, iface.Name+": ")
+		features, err := et.Features(iface.Name)
+		if err == nil {
```

Comment:
maybe wanna log err != nil to the logclient so if it comes up in the wild we can dig it up and try to address it?

Reply:
Good call; added both here and in the `Stats` call